R/utils-fetch.R

```diff
@@ -126,12 +126,9 @@ fetchCorrections <- function(reportObject, seriesCorrName){
 #' @param reportObject the full report data
 #' @param stat 'MIN' or 'MAX' data to select
 fetchMinMaxIVs <- function(reportObject, stat){
-  if(!is.null(stat)){
```

Comment:
Do we not need this null check because we check if it's null somewhere else? I worry that we'll pass in NULL and try and use a function on it.